### PR TITLE
A handful of demonstration changes

### PR DESCRIPTION
**Pull request by richfife**
Per the email discussion we had.

Thanks!

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Can one of the admins verify this patch?

File: interface/src/Application.cpp [hunk 1]
```diff
@@ -5662,6 +5674,9 @@ void Application::updateWindowTitle() const {
     QString title = QString() + (!username.isEmpty() ? username + " @ " : QString())
         + currentPlaceName + connectionStatus + loginStatus + buildVersion;
 
+    if (_windowTag.isSet())
+        title += " " + _windowTag.get();
```
Comment (AndrewMeadows) [inline interface/src/Application.cpp]: As per coding standard [section 4.1.5](https://github.com/highfidelity/hifi/wiki/Coding-Standard#415--the-if-else-class-of-statements-should-have-the-following-form) a one-line `if` block should be wrapped in curly brackets.

File: interface/src/Application.cpp [hunk 2]
```diff
@@ -7651,4 +7675,31 @@ void Application::setAvatarOverrideUrl(const QUrl& url, bool save) {
     _saveAvatarOverrideUrl = save;
 }
 
+void Application::shotCloseUp()
+
+{
+    _scaleMirror = 0.40f;
+}
```
Comment (AndrewMeadows) [inline interface/src/Application.cpp]: As per coding standard [section 4.1.2](https://github.com/highfidelity/hifi/wiki/Coding-Standard#412-use-inline-braces-for-block-layout) the code should be using inline curly braces.

File: interface/src/ui/CameraDistanceDialog.cpp [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+void CameraDistanceDialog::reloadSliders() {
+}
+
+void CameraDistanceDialog::updateDistanceAdjust() {
+}
+
+void CameraDistanceDialog::distanceValueChanged(int value) {
+
+    if(_scaleDistance)
+        *_scaleDistance = static_cast<float>(value) / 10.0f;
```
Comment (AndrewMeadows) [inline interface/src/ui/CameraDistanceDialog.cpp]: As per coding standard [section 4.1.5](https://github.com/highfidelity/hifi/wiki/Coding-Standard#415--the-if-else-class-of-statements-should-have-the-following-form) a one-line `if` block should be wrapped in curly brackets.

File: libraries/script-engine/src/ModelScriptingInterface.cpp [hunk 4]
```diff
@@ -194,6 +194,27 @@ QScriptValue ModelScriptingInterface::getVertex(MeshProxy* meshProxy, int vertex
     return vec3toScriptValue(_modelScriptEngine, pos);
 }
 
+QScriptValue ModelScriptingInterface::getVertexDistance(MeshProxy* meshProxy, int vertexIndex1, int vertexIndex2) {
+
+    QScriptValue v1 = getVertex(meshProxy, vertexIndex1);
+
+    if (v1.isBoolean() && !v1.toBoolean())
+        return v1;
```
Comment (AndrewMeadows) [inline libraries/script-engine/src/ModelScriptingInterface.cpp]: As per coding standard [section 4.1.5](https://github.com/highfidelity/hifi/wiki/Coding-Standard#415--the-if-else-class-of-statements-should-have-the-following-form) a one-line `if` block should be wrapped in curly brackets.

**Comment by richfife** [comment]
Updating with new request

**Pull request closed without merging**